phpunit.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.1/phpunit.xsd"
+         backupGlobals="false"
+         backupStaticAttributes="false"
+         bootstrap="vendor/bootstrap.php"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false"
+         verbose="true"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+        <ini name="zend.enable_gc" value="0"/>
+        <ini name="error_reporting" value="-1"/>
+        <ini name="intl.error_level" value="0"/>
+        <ini name="display_errors" value="On"/>
+    </php>
+
+    <testsuites>
+        <testsuite name="all">
+            <directory>tests</directory>
+        </testsuite>
+    </testsuites>
+    <logging>
+        <log type="coverage-html" target="./coverage/html.html" showUncoveredFiles="false" />
+        <log type="coverage-clover" target="./coverage/clover.xml"/>
+    </logging>
```

Comment:
You should comment these lines too.
Otherwise, the coverage will always be calculate no matter the `COVERAGE_FLAGS`

Reply:
Ok.